activesupport/lib/active_support/cache.rb

```diff
@@ -307,17 +313,30 @@ def fetch(name, options = nil)
       # the cache with the given key, then that data is returned. Otherwise,
       # +nil+ is returned.
       #
+      # As with fetch, the data is only returned if it has not expired per the
```

Comment:
Why mention `fetch`? Also confusing when put together with `Fetches data from the cache` which is how the `read` doc starts.